target/linux/rockchip/image/armv8.mk

```diff
@@ -15,6 +15,15 @@ define Device/friendlyarm_nanopi-r2s
 endef
 TARGET_DEVICES += friendlyarm_nanopi-r2s
 
+define Device/radxa_rock-pi-4
+  DEVICE_VENDOR := Radxa
+  DEVICE_MODEL := Rock Pi 4
```

Comment:
You talk about "Model B v1.3" in the commit message. Is that irrelevant?

Reply:
I just wanted to make it clear on the model version I'm testing.